client/src/app/site/pages/meetings/pages/agenda/modules/topics/modules/topic-poll/components/topic-poll-vote/topic-poll-vote.component.ts

```diff
@@ -64,15 +64,24 @@ export class TopicPollVoteComponent extends BasePollVoteComponent<ViewTopic> imp
         componentServiceCollector: ComponentServiceCollectorService,
         translate: TranslateService
     ) {
-        super(operator, votingService, cd, pollRepo, meetingSettingsService, componentServiceCollector, translate);
+        super(
+            promptService,
+            operator,
+            votingService,
+            cd,
+            pollRepo,
+            meetingSettingsService,
+            componentServiceCollector,
+            translate
+        );
     }
 
     public ngOnInit(): void {
         this.defineVoteOptions();
         this.cd.markForCheck();
     }
 
-    public getActionButtonClass(actions: VoteOption, option: ViewOption, user: ViewUser = this.user): string {
+    protected getActionButtonClass(actions: VoteOption, option: ViewOption, user: ViewUser = this.user): string {
```

Comment:
I believe this is called in the template. Does that still work if this is protected?